### DISCUSSION TITLE
In nested navigator when tabbar hide and show in this process some time it render different tab when try to click header back button

### DISCUSSION THREAD

**Discussion by AmanParikh** [react-navigation/react-navigation#10370] [Q&A]
In app stack navigator inside stack navigator work proper in expo app but when it convert into react-native cli app problem occur
 
TabA- 
          -Stack1
          -stack2
          -stack3
TabB-
          -stack4
TabC-
          -stack5

In app when app navigate to stack navigator we hide bottomTabbar but when we try to goback through header go back button it let us navigate to different tab but in this proccess back button onPress event on render 

Like i go tabA to stack2 then try to goback but it render to TabC  but tabA still on stack2. 

But when we not hidding our tabbar it work properly.

 I use react navigation v6

Is there any solution

**Comment by raajnadar** [comment]
Can you please add a snack repro with steps? So that I can check what is going wrong
